bar-trender-front-end/src/components/ApiFormSubmit.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import { Col, Button, Form, FormGroup, Label, Input } from "reactstrap";
+
+class POSTForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            nombre: "",
+            filtros: {
+                billar: "off",
+                paulanner: "off",
+                reina_mercedes: "off",
+                pueblos: "off",
+                ochentero: "off",
+                arabe: "off",
+            },
+        };
+
+        this.handleTermChange = this.handleTermChange.bind(this);
+        this.handleSearch = this.handleSearch.bind(this);
+        this.handleEnter = this.handleEnter.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleTermChange(e) {
+        this.setState({ [e.target.name]: e.target.value });
+    }
+
+    handleSearch() {
+        // this.props.searchEngine(this.state.term);
+        alert('A form was submitted: ' + JSON.stringify(this.state));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
        alert("A form was submitted: " + JSON.stringify(this.state));
```